### ISSUE TITLE
Error: Unable to create tensor

### ISSUE THREAD

**Issue by spencekim** [xenova/transformers.js#18]
Getting this error when running the embeddings pipeline:

```
Error: Unable to create tensor, you should probably activate truncation and/or padding with 'padding=true' and 'truncation=true' to have batched tensors with the same length.
    at Function._call (tokenizers.js?dd68:928:1)
    at Function.closure [as tokenizer] (utils.js?aa0e:362:44)
    at Function._call (pipelines.js?5d72:46:1)
    at Function._call (pipelines.js?5d72:263:1)
    at closure (utils.js?aa0e:362:44)
    at _callee$ (utils.ts?2b07:20:28)
    at tryCatch (runtime.js?7efe:45:16)
    at Generator.invoke [as _invoke] (runtime.js?7efe:274:1)
    at prototype.<computed> [as next] (runtime.js?7efe:97:1)
    at asyncGeneratorStep (utils.ts?2b07:1:53)
```

I read the source code and saw that padding and truncation are set to true by default, so not sure how this is happening. Any ideas?

**Comment by spencekim** [comment]
I believe the issue could be in pipelines.js on line 48:

```
 async _call(texts) {
        // Run tokenization
        let inputs = this.tokenizer(texts, {
            padding: true,
            truncate: true
        });

        // Run model
        let outputs = await this.model(inputs)

        return [inputs, outputs];
    }
```

Should `truncate` be `truncation` such that it evaluates correctly on line 985 of tokenizers.js?

**Comment by xenova** [comment]
Whoops that does seem to be a typo!

**Comment by spencekim** [comment]
Sounds great, thank you!

**Comment by xenova** [comment]
This should be fixed in the v1.2.5 🎉 (see here https://github.com/xenova/transformers.js/commit/1f0fff6939203489696a14e1d79ec7504f901f24)

Can you check if it works as expected, then I can close the issue :)

**Comment by spencekim** [comment]
Works. Thanks for your help!
